decidim-accountability/app/views/decidim/accountability/results/_home_header.html.erb

```diff
@@ -6,9 +6,9 @@
 
     <div class="small-12 medium-5 columns">
```

Comment:
Can you hide the whole column here? The `.progress-level` element has a white background that looks weird.

Reply:
Oh, of course! Thanks!